protobuf/key_attributes.proto

```diff
@@ -103,44 +100,43 @@ enum AsymmetricEncryptionAlgorithm {
 
 message AsymmetricEncryption {
   AsymmetricEncryptionAlgorithm asymmetric_encryption_algorithm = 1;
-  HashAlgorithm hash_algorithm = 2;
+  Hash hash = 2;
 }
 
-enum MacAlgorithm {
+enum MACAlgorithm {
   HMAC = 0;
   CBC_MAC = 1;
```

Comment:
In the specs it is written that this alg is insecure, might be worth to note it.

Reply:
Hmm, going through the spec again, I think I'll only mark the deprecated ones - there are a whole bunch of warnings around things that are `insecure in many cases`, but it feels like the protobuf contracts is not the place to denote this. Specifying the deprecated ones makes more sense if we actually plan to allow admins to block their usage. We'll need to document this very clearly.

Reply:
It's a bit unfortunate that we can't mark everything that's deprecated as deprecated in the proto files ☹️  

Will try and think of a more explicit way to point things out.